### PR TITLE
Spelling corrected

### PR DESCRIPTION
**Pull request by SachinTitus**
On line 13,  wrong spelling "the4ir" is corrected to "their"

### CODE REVIEW

File: en/3_Open_Source_Software_Use.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ However, although there is no legal difference between these two scenarios, some
...
+Given that separate government departments operate semi-autonomously, there is a possible argument that the different departments constitute different “parties”. However, this interpretation is unlikely. Such an understanding would be incongruent with the general legal understanding of a federal or provincial Crown as being a single legal entity. A 2005 research paper [Legal Issues for the Use of Free and Open Source Software in Government](http://www.austlii.edu.au/au/journals/MelbULawRw/2005/13.html) show that Eben Moglen, general counsel for the FSF, which typically promotes a strong and wide understanding of distribution, takes the view that “[federal Government agencies may share free software without making a ‘distribution’“.
```
Comment (nschonni) [inline en/3_Open_Source_Software_Use.md]: Think this was intentional, but if this changes, the leading `[` should also be removed

**Comment by gcharest** [comment]
I'm on my smartphone at the moment, will review shortly. Or @smellems if he has a chance before me.

Thanks!

File: en/3_Open_Source_Software_Use.md (hunk 1 above)
Comment (smellems) [reply en/3_Open_Source_Software_Use.md]: Yes, I think removing both brackets would be good.  Thanks!

**Comment by smellems** [comment]
Thanks @SachinTitus

**Comment by SachinTitus** [comment]
Thank you @smellems for the opportunity. As a developer who wants to start contributing to open source, I am thankful for the support!

**Pull request merged**